packages/core/test/uiSchema_test.js

```diff
@@ -43,15 +46,26 @@ describe("uiSchema", () => {
           classNames: "class-for-bar another-for-bar",
         },
       },
+      baz: {
+        classNames: "class-for-baz",
+      },
     };
 
     it("should apply custom class names to target widgets", () => {
+      sandbox.stub(console, "warn");
+
       const { node } = createFormComponent({ schema, uiSchema });
-      const [foo, bar] = node.querySelectorAll(".field-string");
+      const [foo, bar, baz] = node.querySelectorAll(".field-string");
 
       expect(foo.classList.contains("class-for-foo")).eql(true);
       expect(bar.classList.contains("class-for-bar")).eql(true);
       expect(bar.classList.contains("another-for-bar")).eql(true);
+      expect(baz.classList.contains("class-for-baz")).eql(true);
+      expect(
+        console.warn.calledWithMatch(
+          /WARNING: 'uiSchema.classNames' is deprecated/
```

Comment:
```suggestion
          /'uiSchema.classNames' is deprecated/
```

Reply:
Will commit this before merging if no other feedback is provided, otherwise will update on another commit